ui/server/src/main/scala/pl/touk/nussknacker/ui/service/ProcessToolbarService.scala

```diff
@@ -92,10 +93,13 @@ private [service] object ToolbarHelper {
   def createProcessToolbarId(config: ProcessToolbarsConfig, process: BaseProcessDetails[_]): String =
     s"${config.uuidCode}-${if(process.isArchived) "archived" else "not-archived"}-${if(process.isSubprocess) "subprocess" else "process"}"
 
-  def fillByProcessData(text: String, process: BaseProcessDetails[_]): String =
+  def fillByProcessData(text: String, process: BaseProcessDetails[_], urlOption: Boolean = false): String = {
+    val processName = if (urlOption) UriUtils.encodeURIComponent(process.name) else process.name
```

Comment:
Is it ok to pass unescaped process name to `name` and `title`? Won't it break if we have names with eg. `"`?

Reply:
Right now at fe w don't escape process name and we don't , so I think it will be ok.